samples/model-builder/create_training_pipeline_text_classification_sample.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+from google.cloud import aiplatform
+
+
+#  [START aiplatform_sdk_create_training_pipeline_text_classification_sample]
+def create_training_pipeline_text_classification_sample(
+    project: str,
+    location: str,
+    display_name: str,
+    dataset_id: int,
+    model_display_name: Optional[str] = None,
```

Comment:
`multi_label: bool=False` should be passed in here

Reply:
Done.